app/scripts/components/boilerplate/view-mode.jsx

```diff
@@ -15,12 +15,12 @@ const ViewMode = React.createClass({
             </Link>
           </li>
           <li>
-            <Link activeClassName="active" to="/region/">
+            <Link activeClassName="active" to="/region/" query={{drilldownField: 'REGION'}}>
```

Comment:
I don't know if we should pass parameter by query or path. 
This could also be done without routes. What do you think is the best option?